app/server/api/user_api.py

```diff
@@ -125,10 +128,15 @@ def get(self, user_id):
     def post(self, user_id):
 
         post_data = request.get_json()
+        preprocess = request.args.get('preprocess', '').lower() == 'true'  #Defaults to false
+
         organisation = g.get('active_organisation')
         if organisation is None:
             return make_response(jsonify({'message': 'Organisation must be set'})), 400
 
+        if preprocess:
```

Comment:
this (plus line 131) is enough to get kobo working, and i think it's much cleaner just to have all POST user requests just go to the POST user endpoint...